src/refactor/packetBreaker.py

```diff
@@ -0,0 +1,8 @@
+import libcsp_py3 as libcsp
+
+class packetBreaker:
+    def __init__(self):
```

Comment:
Unless you plan on extending these, I feel that it would be better if `breakPacket` and `makePacket` were just functions inside of one file (`packetUtils.py`) instead of classes since they don't really carry attributes/data.

Reply:
Done